sim/xcelium/__init__.py

```diff
@@ -0,0 +1,394 @@
+#!/usr/bin/env python3
+
+# HAMMER-VLSI PLUGIN, XCELIUM 
+# Notes: This plugin sets up xrun to execute in a three-step xrun invocation.
+#        This bridges multi-tool direct-invocation and the xrun single-invocation.
+
+
+import os
+import re
+import shutil
+import json
+import datetime
+import io
+import sys
+from typing import Dict, List, Optional, Callable, Tuple
+from multiprocessing import Process
```

Comment:
I see this is not used - what was the intention?

Reply:
Cleaned up, removed callable + process.
Io and sys are used.